test/test.bats

```diff
@@ -37,7 +37,7 @@
 
 @test "trivy fs with securityChecks option" {
   # trivy fs -f json --security-checks=vuln,config -o fs.test .
-  ./entrypoint.sh '-a fs' '-j .' '-b json' '-s vuln,config' '-h fs-scheck.test'
+  ./entrypoint.sh '-a fs' '-j .' '-b json' '-s vuln,config,secret' '-h fs-scheck.test'
```

Comment:
1. Where is `fs-scheck.test`? Did you forgot to check it in?
2. Can we update the repo with a dummy secret and accordingly update the golden file to make an assertion?

Reply:
yeah! @simar7 I forgot to add the data/fs-scheck.test

Reply:
Related (2), I using the trivy repo with the secret option.